### PR TITLE
webui: Fix URL rewrite vulnerability in zend-http component

### PR DESCRIPTION
**Pull request by fbergkemper**
Updates affected zendframework component zend-http
and required component dependencies.

See composer.json for version details.

GHSA: https://github.com/advisories/GHSA-f6p5-76fp-m248

#### Please check

- [x] Short description and the purpose of this PR is present _above this paragraph_
- [x] Your name is present in the AUTHORS file (optional)

If you have any questions or problems, please give a comment in the PR.

### Helpful documentation and best practices

- [Git Workflow](https://docs.bareos.org/DeveloperGuide/gitworkflow.html)
- [Automatic Sourcecode Formatting](https://docs.bareos.org/DeveloperGuide/generaldevel.html#automatic-sourcecode-formatting)
- [Check your commit messages](https://docs.bareos.org/DeveloperGuide/gitworkflow.html#commits)

##### General
- [x] PR name is meaningful
- [x] Purpose of the PR is understood
- [x] Separate commit for this PR in the CHANGELOG.md, PR number referenced is same
- [x] Commit descriptions are understandable and well formatted

##### Source code quality

- [x] Source code changes are understandable
- [x] Variable and function names are meaningful
- [x] Code comments are correct (logically and spelling)
- [x] Required documentation changes are present and part of the PR
- [x] `bareos-check-sources --since-merge` does not report any problems
- [x] `git status` should not report modifications in the source tree after building and testing

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -134,6 +134,7 @@ and since Bareos version 20 this project adheres to [Semantic Versioning](https:
   - webui is NOT affected
   - webui does NOT use the npm server
   - webui does NOT use a user-provided locale string to directly switch moment locale
+- webui: Fix URL rewrite vulnerability in zend-http component [PR #1213] [No known CVE](https://github.com/advisories/GHSA-f6p5-76fp-m248)
```
Comment (pstorz) [inline CHANGELOG.md:137]: We need a "Breaking changes" entry that CentOS7/RHEL7 don't have webui anymore.

File: CMakeLists.txt [hunk 2]
```diff
@@ -89,6 +89,13 @@ if(BUILD_UNIVERSAL_CLIENT)
   set(client-only ON)
 endif()
 
+option(BUILD_WEBUI "Build with WEBUI" ON)
+if(BUILD_WEBUI)
+  set(ENABLE_WEBUI ON)
+else()
+  set(ENABLE_WEBUI OFF)
+endif()
+
```
Comment (pstorz) [inline CMakeLists.txt:98]:
```suggestion
option(ENABLE_WEBUI "Build with WEBUI" ON)
```

**Review by pstorz — CHANGES_REQUESTED** [review]
Good work, but we need to document that bareos-webui is not available for CentOS7/RHEL7 anymore.

File: CHANGELOG.md (hunk 1 above)
Comment (fbergkemper) [reply CHANGELOG.md:138]: Change request applied here https://github.com/bareos/bareos/pull/1213/commits/daa1a2b97bc85b259ca5f3e18d09f6f2a9b2d719

File: CMakeLists.txt (hunk 2 above)
Comment (fbergkemper) [reply CMakeLists.txt]: Change request applied in https://github.com/bareos/bareos/pull/1213/commits/3ee9e618a1ec92c7947bb318f2ef81cabff8f643 and https://github.com/bareos/bareos/pull/1213/commits/11fbdd2ae4c315d7f76c71b0287ad1b3c69eaa21

**Comment by fbergkemper** [comment]
I've put another commit https://github.com/bareos/bareos/pull/1213/commits/8d8125d486e75989f65370b8518accd11f114ad0 on top which updates the system requirements of the webui install section in our documentation.

**Review by pstorz — CHANGES_REQUESTED** [review]
Please squash the "fixup" commits"

**Review by pstorz — APPROVED** [review]

**Pull request merged**